README.md

```diff
@@ -387,16 +387,10 @@ For example, a bucket within the workspace Google project. You can create these
 
 A referenced resource is a cloud resource that is NOT managed by Terra. It exists outside the current workspace
 context. For example, a BigQuery dataset hosted outside of Terra or in another workspace. You can add these with the
-`add-ref` command. The workspace currently supports the following referenced resource: 
-- `gcs-bucket`
-- `gcs-object`
-- `bq-dataset`
-- `bq-table`
-- `git-repo`
+`add-ref` command.
 
 The `check-access` command lets you see whether you have access to a particular resource. This is useful when a
-different user created or added the resource and subsequently shared the workspace with you. `check-access` currently always 
-returns true for `git-repo` reference type because workspace doesn't support authentication to external git services yet.
+different user created or added the resource and subsequently shared the workspace with you.
```

Comment:
Is this maybe a bad merge? It looks like you removed some README additions from another PR.

Reply:
yes, good catch. I don't know how that happened.